tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -315,10 +348,19 @@ def _directory_factory(
             nlb_stack_name = created_directory_stacks.get(region, {}).get("nlb")
             logging.info("Using NLB stack named %s created by another test", nlb_stack_name)
         else:
-            certificate_arn = _generate_certificate()
+            common_name = f"{directory_type}.multiuser.pcluster"
```

Comment:
Nitpick: directory_type could be uppercase/camelcase/whatever, but we want the common name to be lowercase.

Reply:
Common name matching during cert verification should be case insensitivi. that's why this is a minor nitpick. Nevertheless, it looks like there could be situations where the verification is case sensitive and it ciuld cause some headaches :-)

If your tests succeed, let's consider this a nitpick, otherwise, please change.

Reply:
Done